sql2.txt

```diff
@@ -3,12 +3,27 @@ to join the OrderDetails table with the product table.  For TotalCount, you will
 SUM(Quantity) AS TotalCount .  Order by product name.
 You will need to group by product ID.  Paste your SQL statement below.
 
+SELECT p.ProductName, SUM(od.Quantity) AS TotalCount
+FROM Products p
+    JOIN OrderDetails od 
+    ON p.ProductID = od.ProductID
+    GROUP BY p.ProductName, p.ProductID
+    ORDER BY p.ProductID;
```

Comment:
The problem statement says to group by `ProductID` and order by `ProductName`. You were almost there, just keep an eye on these details.